rest_framework/validators.py

```diff
@@ -33,7 +33,16 @@ def qs_filter(queryset, **kwargs):
         return queryset.none()
 
 
-class UniqueValidator(object):
+class ContextBasedValidator(object):
+    """Base class for validators that need a context during evaluation.
+
+    In extension to regular validators their `__call__` method must not only
+    accept a value, but also an instance of a serializer.
+    """
+    pass
```

Comment:
Maybe add the appropriate `def __call__(...)` here which raises `NotImplementedError()`? That's nice for auto-completion and copy/paste, etc.